rbac/management/role/definer.py

```diff
@@ -56,8 +56,8 @@ def _make_role(tenant, data):
             return role
     for access_item in access_list:
         resource_def_list = access_item.pop("resourceDefinitions", [])
-        access_item["perm"] = access_item.pop("permission")
-        access_obj = Access.objects.create(**access_item, role=role)
+        permission, created = Permission.objects.get_or_create(**access_item)
```

Comment:
Do we  need to verify this was created before using it below in creating the new access object?

Reply:
Looks like as long as the we have the permission, it doesn't matter if it is created or not 🤔

Reply:
Keen! No worries then